docs/source/models/causes/opioid_use_disorder/index.rst

```diff
@@ -0,0 +1,821 @@
+
+These parameters are estimated by age, sex, location, and year (1990-2023) and are constrained to be internally consistent through the DisMod-MR 2.1 modeling framework.
+
+**DisMod Prior Settings**
+
+Key modeling assumptions in GBD 2023 include:
+
+- **No incidence or excess mortality before age 15**: Minimum age of onset assumption based on expert feedback and literature
+- **No incidence after age 64**: Modeling assumption based on the very low incidence observed at older ages in data from the European Monitoring Centre for Drugs and Drug Addiction
+- **Remission upper limit of 0.2**: Consistent with limits in the dataset
+- **Country-level covariates**:
+
+  - Age-standardized prevalence of intravenous drug use (IDU) for prevalence
+  - Log-transformed estimates of defined daily doses for statistical purposes (SDDD) of prescribed opioid analgesics (consumption per day per million population) for prevalence
+  - Intravenous drug use as covariate for excess mortality rate (EMR)
+
+**Data Sources**
+
+The GBD 2023 estimates incorporate data from systematic reviews and multiple sources:
+
+- **Direct methods**: Population surveys that ask respondents if they use or are dependent on opioids
+- **Indirect methods**: Multiplier methods, back-projection, and capture-recapture methods to indirectly estimate total number of opioid users
+- **IHME-indirect data**: Created by matching government records of people in opioid substitution therapy with literature sources on percentage of people with opioid dependence in treatment, using spatiotemporal Gaussian process regression (ST-GPR) to estimate coverage
+- **Treatment records**: Number of individuals with opioid dependence in substitution therapy (primarily from government sources)
+- **Published literature**: OUD prevalence, incidence, and treatment coverage estimates
+- **Mortality data**: Vital registration systems and verbal autopsy studies
+
+**Data Adjustment**: Direct survey estimates are adjusted upward using MR-BRT (meta-regression—Bayesian, regularized, trimmed) crosswalk to account for under-ascertainment of highly stigmatized opioid use, based on cross-walks to treatment-based estimates. Surveys tend to underestimate the most harmful and stigmatized forms of illicit drug use.
+
+**Severity Distribution**
+
+GBD 2023 estimates the proportion of individuals with opioid use disorder across three severity levels based on data from the US National Epidemiological Survey on Alcohol and Related Conditions (NESARC) and the Comorbidity and Trauma Study. The severity distribution is: asymptomatic 16% (13-19%), mild 37% (20-55%), and moderate/severe 47% (29-64%).
+
+.. list-table:: Severity Distribution for Opioid Use Disorder
+   :widths: 15 15 45 15
+   :header-rows: 1
+
+   * - Severity Level
+     - Proportion (95% UI)
+     - Lay Description
+     - Disability Weight (95% CI)
+   * - Asymptomatic
+     - 16% (13-19%)
```

Comment:
Asymptomatic seems hard given the list of 7 criteria above. Is this someone who either A) used opioids but can function in daily life fine or B) has actively managed addition?

Reply:
I think these definitions were made by a GBD team, and informed by NESARC survey data.  The relevant NESARC questions are perhaps the following:
<img width="581" height="154" alt="image" src="https://github.com/user-attachments/assets/335be510-9668-4edb-b781-4a7bb88d3387" />
 from Section 3B of [the NESARC](https://www.niaaa.nih.gov/sites/default/files/section%203b_Final_11_18_14.pdf)

Now my big question is how to include "remission" and "asymptomatic severity" in a consistent manner...  I am convinced that the traditional approach to remission in DisMod is not really what we want here.